alfred-telemetry-solr/README.md

```diff
@@ -105,77 +105,77 @@ At the moment following metrics are included, for each core tracked by solr:
 
 | Name                                                                  |
 | ----------------------------------------------------------------------|
-| alfresco_nodes{core=<core>,feature="Approx transactions remaining"}   |
-| alfresco_nodes{core=<core>,feature="TX lag"}                          |
-| alfresco_nodes{core=<core>,feature="Last Index TX Commit Time"}       |
-| alfresco_acls{core=<core>,feature="Approx change sets remaining"}     |
-| alfresco_acls{core=<core>,feature="Change Set Lag"}                   |
-| alfresco_acls{core=<core>,feature="Last Index Change Set Commit Time"}|
+| alfresco.nodes{core=<core>,feature="Approx transactions remaining"}   |
+| alfresco.nodes{core=<core>,feature="TX lag"}                          |
```

Comment:
this needs a different name

Reply:
Have you seen the grafana dashboard(s)?
There some things are split semantically.